manifests/repository.pp

```diff
@@ -4,10 +4,14 @@
       include ::apt
       apt::source { 'puppetlabs':
         location   => 'http://apt.puppetlabs.com',
-        repos      => 'main',
-        key        => '4BD6EC30',
-        key_server => 'pgp.mit.edu',
+        repos      => 'main dependencies',
+        key        => {
+          id     => '47B320EB4C7C375AA9DAE1A01054B7A24BD6EC30',
+          server => 'hkp://pgp.mit.edu:80',
+        },
+        notify     => Exec['apt_update'],
```

Comment:
The notify or the whole code block? I wasn't able to install the puppetserver from scratch, because the package was not found because of the missing apt-get update run. A module should install with defaults on any system and for Ubuntu it won't work for me without this modifications. But I'm open for other solutions.